hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -110,8 +110,16 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
                 source = Diagnostic.Source.Runtime))
         if stderr.nonEmpty then output(s"// Standard Error:\n${stderr}")
       
+      
+      if traceJS.isSet then
+        host.execute("globalThis.Predef.TraceLogger.enabled = true")
+        host.execute("globalThis.Predef.TraceLogger.resetIndent(0)")
```

Comment:
Why not execute both lines in a single call? Splitting them is unnecessarily inefficient.